pom.xml

```diff
@@ -122,6 +122,16 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This dependency is missing:
```
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-engine</artifactId>
            <scope>test</scope>
        </dependency>
```
Otherwise your junit-jupiter tests won't bve executed by surefire plugin.
![image](https://github.com/user-attachments/assets/53bd2dbf-5fbc-400e-8242-b0c1e09c486c)